### ISSUE TITLE
Update in line with Matomo v6

### ISSUE THREAD

**Issue by gligorkot** [etienne-martin/device-detector-js#157]
@etienne-martin https://www.npmjs.com/package/device-detector-js is quite popular - getting about 100k downloads per week. It would be good to make a long term plan on how to keep it up to date with the Matomo PHP version.

I made a fork yesterday and have a local version updated to the latest v5.0.5, however v6 of the PHP version introduced improved detection using client hints and this is quite a bit of effort. Can we try and see if there's a way to involve more people from the community to give this library some love so we can get it updated? I'm happy to chip in and also do code reviews if need be, but not sure I can commit to fully maintaining it.

It would be nice to add something about this on the Readme page so that people visiting the npm page can get notified that this library needs contributors to get updated.

**Comment by mattiLeBlanc** [comment]
Happy to help too if I can. 
Just found this lib and it is exactly what I need.

**Comment by whimsicaldreamer** [comment]
@mattiLeBlanc  @gligorkot Maybe you can consider using https://github.com/sanchezzzhak/node-device-detector as an alternative for the time being. It is maintained by one of the contributors of matomo device detector.

**Comment by gligorkot** [comment]
> @mattiLeBlanc  @gligorkot Maybe you can consider using https://github.com/sanchezzzhak/node-device-detector as an alternative for the time being. It is maintained by one of the contributors of matomo device detector.

Yeah, in the end that's what we went with. Thanks for dropping a not here 🙏
